### ISSUE TITLE
Pagination does not work with many pages

### ISSUE THREAD

**Issue by andreas83** [BookingSync/ember-cli-jsonapi-pagination#1]
My [RSS API](http://rssly.codejungle.org/api/1/items/) have many many pages, around 365211, which breaks the functionality (browser get stuck). 
I use following code to get the total pages:

```
 totalPages: Ember.computed('size', 'number', 'model.[]', function() {
    return this.get('model.meta.pagination.pages');
  })
```

**Comment by Azdaroth** [comment]
Hey @andreas83,

wow, 365211 pages... I think this might be causing serious performance issue, current way of generating paginator is quite inefficient in that regard and it really creates 365211  Ember Objects under the hood, which is absolutely not needed. I will try to improve it this week and release a new version. Will let you know once done.

**Comment by andreas83** [comment]
wow super fast response, thank you!

**Comment by Azdaroth** [comment]
Hey @andreas83 !

I took a look at it and already submitted a PR for that: https://github.com/BookingSync/ember-cli-jsonapi-pagination/pull/2

I may want to refactor some parts or check if some tests are not missing, but it should already improve the performance, quite substantially. 

Let me know how it works, I'll probably release a new version tomorrow or on Sunday.

**Comment by andreas83** [comment]
works like a charme !!! :+1:

**Comment by Azdaroth** [comment]
@andreas83 awesome! 🎉 Expect the new version quite soon :)

**Comment by Azdaroth** [comment]
Closed by https://github.com/BookingSync/ember-cli-jsonapi-pagination/pull/2

**Comment by Azdaroth** [comment]
@andreas83 Just released 0.0.2 version 🎉
